src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -324,7 +337,30 @@ private int ComputeRhCorElementType(TypeDesc type)
             Debug.Assert(false, "Primitive type value expected.");
             return 0;
         }
-        
+
+        private static bool TypeRequiresAlign8(TypeDesc type)
```

Comment:
Maybe call this `ComputeRequiresAlign8` to make it look like `ComputeRhCorElementType`?